sky/packages/sky/lib/base/lerp.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:sky';
```

Comment:
Yeah.  I moved lerpNum into dart:sky as a free function and moved the rest into statics in #952.  We can try to get num.lerp as a separate step.  :)